_locales/en/messages.json

```diff
@@ -414,6 +415,9 @@
   "dialogClearAllDataDeletionFailedTitleQuestion": "Do you want to delete data from just this device?",
   "dialogClearAllDataDeletionFailedMultiple": "Data not deleted by those Service Nodes: $snodes$",
   "dialogClearAllDataDeletionQuestion": "Would you like to clear this device only, or delete your data from the network as well?",
+  "clearDevice": "Clear Device",
+  "tryAgain": "Try Again",
+  "areYouSureClearDevice": "Are you sure you want to delete your device?",
```

Comment:
This should say clear device not delete device as in Figma.

Reply:
amended